doc/mgmt/OpenConfig_System.md

```diff
@@ -0,0 +1,556 @@
+    * aaa
+    * processes
+    * messages
+
+### 1.1.2 Configuration and Management Requirements
+The System configuration/management can be done via REST and gNMI. The implementation will return an error if configuration is not allowed due to misconfiguration or un-supported node is accessed.
+
+### 1.1.3 Scalability Requirements
+NA
+
+## 1.2 Design Overview
+### 1.2.1 Basic Approach
+SONiC already supports framework for Get, Patch and Delete via REST and gNMI. This feature adds support for OpenConfig based YANG models using transformer based implementation for System features.
+
+### 1.2.2 Container
+The code changes for this feature are part of *Management Framework* container which includes the REST server and *gnmi* container for gNMI support in *sonic-mgmt-common* repository.
+
+# 2 Functionality
+## 2.1 Target Deployment Use Cases
+1. REST client through which the user can perform POST, PUT, PATCH, DELETE, GET operations on the supported YANG paths.
+2. gNMI client with support for capabilities, get, set and subscribe based on the supported YANG models.
+
+# 3 Design
+## 3.1 Overview
+This HLD design is in line with the [https://github.com/sonic-net/SONiC/blob/master/doc/mgmt/Management%20Framework.md]
+
+## 3.2 DB Changes
+### 3.2.1 CONFIG DB
+There are no changes to CONFIG DB schema definition.  
+For software-version, new table will be added, namely VERSIONS|SOFTWARE.
+
+### 3.2.2 APP DB
+There are no changes to APP DB schema definition.
+
+### 3.2.3 STATE DB
+There are no changes to STATE DB schema definition.
+
+### 3.2.4 ASIC DB
+There are no changes to ASIC DB schema definition.
+
+### 3.2.5 COUNTER DB
+There are no changes to COUNTER DB schema definition.
+
+## 3.3 User Interface
+### 3.3.1 Data Models
+Openconfig-system.yang (revision 2.1.0) and its submodules will be used as user interfacing models.  
+We are updating openconfig-system yang version (0.7.0) in sonic with latest available openconfig version (revision 2.1.0), to have all the latest changes from community.  
+Community PR [https://github.com/sonic-net/sonic-mgmt-common/pull/147]  
+   * In this PR openconfig-system.yang and its submodules are updated to latest version available
+   * Reference to openconfig-network-instance model is commented, as this module is yet to be supported in SONiC
+
+Main changes in the latest openconfig versions are:
+ * Feature wise major changes -
+    * system/state -> up-time, software-version, last-configuration-timestamp nodes are added.
+    * ntp -> source-address moved to per server list from global container. Also network-instance is included per server. NTP Key reference is added in server.
+    * grpc-server -> Restructured completely, multiple server provision is added.
+    * logging -> For remote-server, network-instance support is added. Files & VTY containers are added newly.
+    * memory -> used and free leaves are added.
+    * process -> uptime node is removed.
+
+ * New features added in latest version -
```

Comment:
Are you going to support new features listed below?

Reply:
No, we are not supporting these containers

Just added here a list of new containers added in latest version of openconfig-system.yang for reference